dandischema/datacite/__init__.py

```diff
@@ -98,7 +101,13 @@ def to_datacite(
     attributes["descriptions"] = [
         {"description": meta.description, "descriptionType": "Abstract"}
     ]
-    attributes["publisher"] = "DANDI Archive"
+    attributes["publisher"] = {
+        "name": "DANDI Archive",
+        "schemeUri": "https://scicrunch.org/resolver/",
+        "publisherIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017571",
+        "publisherIdentifierScheme": "RRID",
```

Comment:
RRID technically isn't supported yet, though it should be very soon (when 4.6 is released). It doesn't fail out on the jsonschema because of a bug https://github.com/inveniosoftware/datacite/issues/103

It may work right now because DataCite is rolling out 4.6 support, but is worth double checking it goes through to Fabrica.

Reply:
We checked, nothing in 4.6 PR seems to suggest for it to become a controlled dictionary, just a "string" ATM, so  I  think we should keep it.

FWIW -- I did open and close
- https://github.com/datacite/schema/issues/150

I guess we could have switched to https://www.re3data.org/repository/r3d100013638  which also points to the RRID.  But it also shows that may be here it could be `publisherIdentifiers` list of records? (I don't want even to suggest that "officially" ;-) )

Reply:
Ah, you're right. I got messed up by the examples and that RRID is going to be supported as a related identifier type in 4.6. I was just on the ROR call today, where they showed the data in this field is a mess. But that's that happens without a controlled list.

Reply:
I think there is nothing more todo here, marking resolved, please reopen/comment if I was wrong